Source/WebKit/UIProcess/Extensions/WebExtensionController.h

```diff
@@ -96,7 +98,12 @@ class WebExtensionController : public API::ObjectImpl<API::Object::Type::WebExte
     void removePage(WebPageProxy&);
 
     WebPageProxySet allPages() const { return m_pages; }
-    UserContentControllerProxySet allUserContentControllers() const { return m_userContentControllers; }
+
+    // Includes private browsing content controllers.
+    UserContentControllerProxySet allUserContentControllers() const { return m_allUserContentControllers; }
+
+    // Excludes private browsing content controllers.
+    UserContentControllerProxySet normalUserContentControllers() const { return m_normalUserContentControllers; }
```

Comment:
I'm not sure what better name to use, but `normal` could be a bit more descriptive. It's hard to know that the opposite of normal is normal + private.

Reply:
Yeah I struggled with this too. Will think on it and change it if I think of a better name later.

Reply:
`nonPrivateContentControllers`? or `contentControllersNotUsedInPirvateBrowsing` ... neither are great tbh. Or you could do the reverse and have `allUserContentControllers` and then a `privateUserContentControllers`?

Reply:
I went with:

```cpp
    // Includes both public and private browsing content controllers.
    UserContentControllerProxySet allUserContentControllers() const { return m_allUserContentControllers; }

    // Excludes private browsing content controllers.
    UserContentControllerProxySet allNonPrivateUserContentControllers() const { return m_allNonPrivateUserContentControllers; }
```